code/src/components/Playlists.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import stretchgoal from "../stretchgoal.json";
+
+// Component for the sidebar with the playlists, includes playlist name and image with play icon on top
+
+export const Playlists = () => {
+    return (
+        <aside className="playlists">
+            <h2 className="playlist-header">Top Playlists</h2>
+            {stretchgoal.playlists.items.map(playlist => {
```

Comment:
It would be more readable if you use `stretchgoal.playlists.items.map(playlist => ( ) )` to avoid using `return` nested in another `return`